src/main/java/gr/cytech/chatreminderbot/rest/controlCases/CaseShowVersion.java

```diff
@@ -0,0 +1,28 @@
+package gr.cytech.chatreminderbot.rest.controlCases;
+
+import gr.cytech.chatreminderbot.rest.message.Request;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Properties;
+
+public class CaseShowVersion {
+    private static final Logger logger = LoggerFactory.getLogger(CaseShowReminders.class);
+
+    private Request request;
+
+    public Request getRequest() {
+        return request;
+    }
+
+    public String showVersion(Request request) throws Exception {
```

Comment:
If I understand correctly, whatever error you might get, you just throw it without ever catching it.
This will finally get handled by the container, but your bot won't respond anything, right?
It would be **much** better if you handled any errors right where it makes sense. E.g. now, inside your showVersion, you should catch any Exceptions and handle them in it, such as showing `Internal error while finding current version`